rollup.config.js

```diff
@@ -5,6 +5,7 @@ import { babel } from "@rollup/plugin-babel";
 import { terser } from "rollup-plugin-terser";
 import filesize from "rollup-plugin-filesize";
 import { visualizer } from "rollup-plugin-visualizer";
+import typescript from "@rollup/plugin-typescript";
```

Comment:
No need to change this now, but in the future we might look into swapping rollup out for Vite (which uses rollup under the hood), just because the plugin/config is a bit less verbose. Maybe something for "future us".